flexmeasures/data/models/migration_utils.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from typing import List
+
+from flexmeasures.data import db
+
+
+def copy_old_sensor_attributes(
```

Comment:
I'd like some comments, at least to the effect of why these functions are here (the data model migration) and when we might get rid of them again.